packages/broker/test/integration/plugins/operator/MaintainOperatorValueHelper.test.ts

```diff
@@ -0,0 +1,51 @@
+import { Wallet } from 'ethers'
+import { Contract } from '@ethersproject/contracts'
+import { Provider } from '@ethersproject/providers'
+
+import { operatorFactoryABI, OperatorFactory } from '@streamr/network-contracts'
+import { config } from '@streamr/config'
+
+import { STREAMR_DOCKER_DEV_HOST } from '../../../utils'
+import { setupOperatorContract, SetupOperatorOpts } from './setupOperatorContract'
+import { getProvider } from './smartContractUtils'
+import { MaintainOperatorValueHelper } from '../../../../src/plugins/operator/MaintainOperatorValueHelper'
+
+const chainConfig = config.dev2
+const theGraphUrl = `http://${STREAMR_DOCKER_DEV_HOST}:8800/subgraphs/name/streamr-dev/network-subgraphs`
+
+const STREAM_CREATION_KEY = '0xb1abdb742d3924a45b0a54f780f0f21b9d9283b231a0a0b35ce5e455fa5375e7'
```

Comment:
Maybe better name if we don't create streams with this key?

Reply:
Updated